packages/sdk/src/hooks/useCredentialRecordById.tsx

```diff
@@ -0,0 +1,10 @@
+import { useCredentialRecords } from './useCredentialRecords'
+
+export const useCredentialRecordById = (id: string) => {
```

Comment:
I think for all these hooks we need to update it to use the for display variant by default (which also includes the record i think?)

Reply:
the `useCredentials` now returns display information, and the `useCredentialRecords` only the record. `useCredentials` will be a public hook and `useCredentialRecords` will be an internal hook